source/configure/database-configuration-settings.rst

```diff
@@ -408,3 +408,25 @@ Replica lag settings
 |                                                                                                                                           |
 |   select member_id, count_transactions_remote_in_applier_queue FROM performance_schema.replication_group_member_stats where member_id=<>  |
 +--------------------------------------------------------+----------------------------------------------------------------------------------+
+
+.. config:setting:: database-replicamonitorintervalseconds
+  :displayname: Replica monitor interval (Database)
+  :systemconsole: N/A
+  :configjson: .SqlSettings.ReplicaMonitorIntervalSeconds
+  :environment: MM_SQLSETTINGS_REPLICAMONITORINTERVALSECONDS
+
+  Specifies how frequently unhealthy replicas will be monitored for liveness check. Mattermost will dynamically choose a replica if it's alive.
+
+Replica monitor interval (seconds)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
```

Comment:
@agnivade - Is this config setting available for all plans and both self-hosted and cloud deployments? (or is this intended as a paid feature via Professional or Enterprise)?

Reply:
It cannot be modified in cloud. But for self-hosted, it's available in all editions.

Reply:
Thanks, @agnivade! Is the default used in Cloud deployments?

Reply:
I am not sure, you'll have to ask the SRE team on that.

Reply:
@nickmisasi - Can you confirm whether Cloud deployments are using this config setting, and the value being used, if not default, please?